### ISSUE TITLE
Bridge connection - Pliers not showing in JEI

### ISSUE THREAD

**Issue by LucioneWolff** [sketchmacaw/MacawsModsIssues#299]
Macaw's bridge, minecraft 1.20, forge 46.0.14

Connecting bridges wider than 1x
There's no issue when building bridges with a width of 1x, but problems arise when this value is greater than 1, for example, a width of 2x, 3x, and so on. The textures and collisions end up being connected, making it difficult to access the bridge naturally.

![image](https://github.com/sketchmacaw/MacawsModsIssues/assets/66290745/88db419e-fb9c-41d6-a682-5afa81eff917)
![image](https://github.com/sketchmacaw/MacawsModsIssues/assets/66290745/618e689f-a290-426b-96e0-beaf35146936)

Additionally, I have tried using the stairs or supports from the "Macaw's Bridge" mod to see if it made any difference, but unfortunately, no matter what I do, the textures and hitbox.

Thank you very much for your attention and for the mod! :D
No problem at all! I apologize for my English, I used a translator.

**Comment by sketchmacaw** [comment]
You can craft the item "Pliers" and right click where you want to remove the connection/railing :)

**Comment by sketchmacaw** [comment]
https://youtu.be/jwQf46NnCtE?t=161 Here you can see how it works!

**Comment by quikenb** [comment]
Just to add onto this. I think the issue mainly comes from the pliers not showing in JEI/REI.

**Comment by sketchmacaw** [comment]
Ah thanks, didn't knew that

**Comment by quikenb** [comment]
So, let me get you some more info my man.
I have looked through a decompile and the recipe is definitely in your code, unlike someone that accidentally deleted it..... (JaisDk :3). It has been a common issue with JEI that recipes do not show or the item shows but the recipe doesn't, although crafting the recipe totally works. Just to clarify. The recipe works just pliers do not show in jei. I have also tried pliers do not show in the creative item menu either.
Running ATM9 Alpha v0.0.10 MC 1.20.1 Forge 47.0.14
Following Macaw's mods and versions
    Bridges - 2.1.0
    Bridges BOP - 1.20-1.0
    Fences BOP - 1.20-1.0
    Fences & Walls - 1.0.7
    Painting - 1.0.4
    Roofs - 2.2.4
    Roofs BOP - 1.20-1.0
